src/dotnet-install.sh

```diff
@@ -618,41 +618,89 @@ get_specific_product_version() {
     if [ $# -gt 2  ]; then
         local package_download_link="$3"
     fi
-    local specific_product_version=$specific_version
+    local specific_product_version=null
+
+    local download_links=($(get_specific_product_version_url "$azure_feed" "$specific_version" true "$package_download_link")
+        $(get_specific_product_version_url "$azure_feed" "$specific_version" false "$package_download_link"))
+
+    for download_link in "${download_links[@]}"
+    do
+        say_verbose "Checking for the existence of $download_link"
+
+        if machine_has "curl"
```

Comment:
I'm still complaining about this logic :) 
Would it be easier to modify download function to do it?

Reply:
I'm hesitant to make such changes as I'm not experienced at all with .sh scripts and we are under a time constraint.